File: MonitoreoCNET/MonitoreoCNET/Models/Servicios/Servicio.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MonitoreoCNET.Models.Servicios
{
    [Table("servicio")]
    public class Servicio
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Column("nombre")]
        public string Nombre { get; set; }

        public Servicio(string nombre)
        {
            Nombre = nombre;
        }

        public Servicio()
        {
        }
    }
}

File: monitoreo/src/main/java/models/entidades/servicios/servicio/Servicio.java
package models.entidades.servicios.servicio;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "servicio")
public class Servicio {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Getter @Setter
  private int id;

  @Getter @Setter
  @Column(name = "nombre")
  private String nombre;

  public Servicio(String nombre) {
    this.nombre = nombre;
  }

  public Servicio() {
  }
}
